patches/chrome-android-java-src-org-chromium-chrome-browser-ChromeTabbedActivity.java.patch

```diff
@@ -11,3 +11,47 @@ index be8466ad87791880f1f3c4ea61db11c466eb3356..9d977486ffffda3c6545d974c1f4e8a2
      /**
       * The results of a system back press action.
       */
+@@ -965,13 +965,13 @@ public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMo
+     }
+ 
+     @Override
+-    public ChromeTabCreator getTabCreator(boolean incognito) {
+-        return (ChromeTabCreator) super.getTabCreator(incognito);
++    public org.chromium.chrome.browser.tabmodel.BraveTabCreator getTabCreator(boolean incognito) {
++        return (org.chromium.chrome.browser.tabmodel.BraveTabCreator) super.getTabCreator(incognito);
+     }
+ 
+     @Override
+-    public ChromeTabCreator getCurrentTabCreator() {
+-        return (ChromeTabCreator) super.getCurrentTabCreator();
++    public org.chromium.chrome.browser.tabmodel.BraveTabCreator getCurrentTabCreator() {
++        return (org.chromium.chrome.browser.tabmodel.BraveTabCreator) super.getCurrentTabCreator();
+     }
+ 
+     @Override
+@@ -1659,13 +1659,13 @@ public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMo
+     }
+ 
+     @Override
+-    protected Pair<ChromeTabCreator, ChromeTabCreator> createTabCreators() {
++    protected Pair<org.chromium.chrome.browser.tabmodel.BraveTabCreator, org.chromium.chrome.browser.tabmodel.BraveTabCreator> createTabCreators() {
+         Supplier<TabDelegateFactory> tabDelegateFactorySupplier = () -> {
+             return new TabbedModeTabDelegateFactory(this);
+         };
+-        return Pair.create(new ChromeTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
++        return Pair.create(new org.chromium.chrome.browser.tabmodel.BraveTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
```

Comment:
this patch is correct to change the class that is actually created, but the patch for the method signature is not needed